doc/api/fs.md

```diff
@@ -1897,6 +1897,10 @@ Any specified file descriptor has to support reading.
 *Note*: If a file descriptor is specified as the `path`, it will not be closed
 automatically.
 
+*Note*: `fs.readFile()` reads the entire file in a single threadpool request.
+To minimize threadpool task variation, prefer the partitioned APIs `fs.read()`
+and `ReadStream` when reading files as part of fulfilling a client request.
```

Comment:
Nit: using `fs.createReadStream()` instead of `ReadStream` here may be clearer for most users.

Reply:
Addressed in 6789de6